documentation/_include/podman.adoc

```diff
@@ -428,7 +428,7 @@ UID          PID    PPID  C STIME TTY          TIME CMD
 root           1       0  0 17:23 ?        00:00:00 /bin/ps -ef
 ----
 
-Notice that all the process belonging to host itself are absent.  The programs running in the container's namespace are isolated from the rest of the host.  From the container's perspective, the process in the container is the only process running.
+Notice all the process belonging to the host itself are absent.  The programs running in the container's namespace are isolated from the rest of the host.  From the container's perspective, the process in the container is the only process running.
```

Comment:
Notice that all of the processes belonging to the host itself are absent.

Reply:
And I missed updating process to the plural processes.
Intended to be "Notice all the processes belonging to the host itself are absent."